tests/integration/kubernetes/runtimeclass_workloads/pod-confidential-unencrypted.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+kind: Service
+apiVersion: v1
+metadata:
+  name: "confidential-unencrypted"
+spec:
+  selector:
+    app: "confidential-unencrypted"
+  ports:
+  - port: 22
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: "confidential-unencrypted"
+spec:
+  selector:
+    matchLabels:
+      app: "confidential-unencrypted"
+  template:
+    metadata:
+      labels:
+        app: "confidential-unencrypted"
+    spec:
+      runtimeClassName: kata
```

Comment:
Not really.  This is taken care as part of the tests setup.

If we change to something else, which I am not against, I'd like to have all the tests changed together to avoid discrepancy.